### ISSUE TITLE
Implementation code for f (i, k) not found

### ISSUE THREAD

**Issue by Bonnie768** [zhengzhang01/Pixel-GS#12]
Hello, I couldn't find the implementation code for the added module f (i, k). Can you tell me where it is?
<img width="398" alt="微信图片_20241211204319" src="https://github.com/user-attachments/assets/e9bfc81b-ab6f-4a68-90b0-87299b1e8849">

**Comment by lyq0524** [comment]
Me too. Have you found it now?

**Comment by YeJiangnan1029** [comment]
I find it. When we call loss.backward(), the scaling factor f(i,j) has been applied to dL_dxy by getting z-depth in backward process. Here you can see:
https://github.com/zhengzhang01/Pixel-GS/blob/05c9d557aa3749bb1ff05e2627943afdd9c18ffd/submodules/diff-gaussian-rasterization/diff_gaussian_rasterization/__init__.py#L144
